### PR TITLE
Software materials

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Review by brianhlin — COMMENTED** [review]
Hey Christina, you'll also need to add the exercise pages to `mkdocs.yml` in the root of this repo and you probably want to add links to the exercises and talks in `docs/materials/index.md`

**Comment by ChristinaLK** [comment]
Links added @brianhlin

**Comment by brianhlin** [comment]
CI looks like it's failing due to a broken link: https://travis-ci.org/opensciencegrid/user-school-2019/builds/556893222#L542

**Comment by brianhlin** [comment]
Looks like another one: https://travis-ci.org/opensciencegrid/user-school-2019/builds/556905182#L541-L542

**Comment by ChristinaLK** [comment]
@brianhlin I think I've fixed all the links but travis is reporting a link I've already changed. is it possible to re-kick off the build?

File: docs/materials/day3/part1-ex2-matlab.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+		:::console
+		tar -tf r2015b.tar.gz
+
+1. The output of the previous command is a list of all the files in the tar.gz file. What is the name of the first folder of the path for each file? This is the name of the runtime directory, and the argument you should pass to `run_matrix.sh`.  
+
+Submitting the Job
+------------------
+
+1.  Copy an existing submit file into your current directory. The submit file we used for the [Open Bugs](part3-ex4-prepackaged.md) example would be a good candidate, as that example also used a wrapper script.
```
Comment (brianhlin) [inline docs/materials/day3/part1-ex2-matlab.md]: @ChristinaLK I was really confused about the broken link, too. It looks like you've moved the 'prepackaged' exercise to Tuesday and this matlab exercise is in Wednesday so we need to specify the path. If you accept this suggestion, I think it should fix everything.

```suggestion
1.  Copy an existing submit file into your current directory. The submit file we used for the [Open Bugs](/materials/day2/part3-ex4-prepackaged.md) example would be a good candidate, as that example also used a wrapper script. 
```

**Review by brianhlin — APPROVED** [review]
LGTM!

**Comment by ChristinaLK** [comment]
thanks for your patience!

**Comment by brianhlin** [comment]
Likewise! I wish the CI would dump out the list of all broken links instead of having to go through them one-by-one D:

**Pull request merged**